### ISSUE TITLE
Inspiration in newer papers about double array tries

### ISSUE THREAD

**Issue by dumblob** [hankcs/AhoCorasickDoubleArrayTrie#49]
Is this based on [Speeding Up Double-Array Trie Construction for String Matching](https://link.springer.com/chapter/10.1007/978-3-642-35795-4_72 ) or something else?

**Comment by hankcs** [comment]
No, although I wrote this after the paper you mentioned, I did not read it and I can't tell if the algorithms are the same.

**Comment by dumblob** [comment]
Have you documented somewhere which approach you've taken for modifications?

**Comment by hankcs** [comment]
No, but I wrote a blog post on it:https://www.hankcs.com/program/algorithm/aho-corasick-double-array-trie.html

You may need a translator.

**Comment by dumblob** [comment]
Nice! I've read a translated version (Google translate) and find it interesting.

Any plans to rewrite it in a closer-to-metal language ([V](https://vlang.io/ ) or Rust or C++ or C or Go etc.)?

**Comment by hankcs** [comment]
It's a good idea but I don't have time now. I'm pretty sure there is a go implementation somewhere on GitHub.
